src/_posts/databases/mysql/2000-01-01-overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Scalingo for MySQL® Overview
+nav: Overview
+modified_at: 2024-03-21 12:00:00
+tags: database mysql addon
+index: 1
+---
+
+## MySQL®
+
+MySQL® is a free and open-source relational database management system. It is
+known for its reliability, ease of use, and performance, making it one of the
+most popular database systems in the world. It is well-suited for a wide range
+of applications, from small-scale websites to large-scale entreprise solutions.
+
+
+## Scalingo for MySQL®
+
+Scalingo for MySQL® is our official MySQL® addon. [We have several
+plans available](https://scalingo.com/databases/mysql#database-compare),
+which you can chose of depending on your needs and budget. As always, a free
+plan is available for you to test our offer.
+
+### Features
+
+All our plans are fully managed. They allow to scale your database to another
+plan whenever you want.
+
+They also include access to metrics and logs, allowing for quick and easy
+monitoring and auditing. For more information regarding this topic, please [see
+our dedicated page]({% post_url databases/mysql/2000-01-01-monitoring %}).
+
+From the security point of view, they all come with a TLS connection and
+encryption at rest.
+
+In addition to the above, Starter and Business plans include daily automatic
+backups. All backups are encrypted and stored externally. For more information
+regarding this topic, please [see our dedicated page]({% post_url databases/mysql/2000-01-01-backing-up %}).
+
+For the most demanding environments, our Business plans include a fully-managed
+InnoDB cluster made of 3 MySQL® nodes in a private network and 2 MySQL® Routers
+(one leader and one failover) as entrypoint to your cluster private network. The
+communications in the cluster are encrypted. Chosing such a plan drastically
+lowers the risk of unavailability, be it during a maintenance operation
+(database upgrade, for example) or because of a platform issue.
+
+### Available Version
+
+The latest version of MySQL® available is **`8.0.35-1`**.
+
+The default version when attaching a Scalingo for MySQL® addon to your
```

Comment:
Not anymore

Reply:
Default version is now **`8.0.35-1`**

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/2501/commits/a9e8976937c82cb5438d7f761233bc2a91ace92c